presto-docs/src/main/sphinx/sql/set-session.rst

```diff
@@ -13,14 +13,36 @@ Synopsis
 Description
 -----------
 
-Set a session property value.
+Set a session property value or a catalog session property.
+
+Catalog session properties use the name of a catalog as a prefix in front of a
+session property supported by a connector.
 
 Examples
 --------
 
+The following example enables optimized hash generation.
```

Comment:
Add a colon at the end. This can then be simplified, as SQL is the default language:
```
The following example sets a system session property to enable optimized hash generation::

    SET SESSION optimize_hash_generation = true;

Reply:
I left it as it was originally .. but I have updated it now as requested. Although I am not a fan of the :: usage since it seem to confuse other writers